climada/hazard/centroids/centr.py

```diff
@@ -331,16 +331,21 @@ def from_pnt_bounds(cls, points_bounds, res, crs=DEF_CRS):
             }
         )
 
-    def append(self, centr):
-        """Append Centroids
+    def append(self, *centr):
+        """Append Centroids to the current centroid object for concatenation.
+
+        This method check that all centroids use the same CRS, append the list of centroids to
+        the initial Centroid object and eventually concatenate them to create a single centroid
+        object with the union of all centroids.
 
         Note that the result might contain duplicate points if the object to append has an overlap
-        with the current object.
+        with the current object. Remove duplicates by either using :py:meth:`union`
+        or calling :py:meth:`remove_duplicate_points` after appending.
 
         Parameters
         ----------
-        centr : Centroids
-            Centroids to append. The centroids need to have the same CRS.
+        centr : list
+            List of Centroids to append. The centroids need to have the same CRS.
```

Comment:
since `centr` is given as a `*args`, it does not have to be a list but can be several centroids, no? Formulated differently, can this method be called these two ways with equivalent results or does only one way work?
```
centr.append(new_centrs1, new_centrs2)
```
```
centr.append([new_centrs1, new_centrs2])
```

Reply:
Hi Benoit, I think you are right, I will update the docs strings, it shouldn't be a list.